### ISSUE TITLE
No OpenID Server found.

### ISSUE THREAD

**Issue by wortel538** [SmItH197/SteamAuthentication#54]
If I use this, I get the error "No OpenID Server found at http://steamcommunity.com/openid"
How to fix it, It just replaces the button with this whenever you press on it, after 1 minute waiting..

Thanks to all the makers, You are amazing! It worked before..

**Comment by BlackCetha** [comment]
If you browse to that URL directly, you are clearly able to download the openid.xml. Maybe the Steam servers where down while you tested or your server doesnt have access to URLs, maybe caused by hoster limitation.

**Comment by SmItH197** [comment]
open up steamauth.php and go to line 23 and change `http://steamcommunity.com/openid` to `https://steamcommunity.com/openid`.

**Comment by wortel538** [comment]
^^ I changed line 23, Now it succesfull goes to steam-login, then I go to the page back (not login page) and says "No OpenID Server found at http://steamcommunity.com/openid/id/76561198043436466"

**Comment by SmItH197** [comment]
Could you create a new php file on your web host and put the following code in:
```php
<?php
if( ini_get('allow_url_fopen') ) {
   echo "enabled";
} else {
    echo "disabled";
}
?>
```
And visit it in your browser and tell me if it says `enabled` or `disabled`?

**Comment by wortel538** [comment]
enabled

**Comment by SmItH197** [comment]
Hi, what hosting do you use? I will try and replicate the problem

**Comment by wortel538** [comment]
Try after a few minutes again.. I got alot too, I want to switch over only when I got everything ready, Or do you know an other free hosting

**Comment by wortel538** [comment]
Okay, Found the problem, It was just the provider. Thanks for all help, Cya! Keep the work up!

**Comment by wortel538** [comment]
Closed issue #54
